### ISSUE TITLE
The problem about self.rel_rec and self.rel_send 

### ISSUE THREAD

**Issue by 1zgh** [limaosen0/AS-GCN#17]
rel_rec = self.rel_rec.cuda(gpu_id)
		rel_send = self.rel_send.cuda(gpu_id)

		self.logits = self.encoder(x, rel_rec, rel_send) 

 The dim of rel_rec and rel_send  is 【600，25】,but I can not understand what they mean about human skeleton.   AIM network is a little hard to understand. I would appreciate it if you could elaborate on your ideas.  Thanks!!

**Comment by czhaneva** [comment]
I have read the code too, it's difficult to understanding the encoder-decoder as well.

**Comment by Liu0114** [comment]
Hello,Are you understand the network of AIM?
